### ISSUE TITLE
TrailingWhitespace reports whitespaces in raw strings

### ISSUE THREAD

**Issue by lwasyl** [arturbosch/detekt#1966]
## Expected Behavior
Raw strings are allowed trailing whitespace

## Observed Behavior
Trailing whitespace in raw string is reported as an error

## Steps to Reproduce
Have a trailing whitespace in raw string

## Context
I believe raw strings should be exempt from such code style rules, since it might be inteded that there's trailing space before line break

## Your Environment
* Version of detekt used: 1.0.1

**Comment by cortinico** [comment]
> Have a trailing string in raw string

Just to clarify, what exactly do you mean with _raw string_? Do you mean multi-line strings (`"""`)?

**Comment by lwasyl** [comment]
Yes, for example
```kotlin
val text = """
this has whitespace after dot.     
"""
```

**Comment by schalkms** [comment]
I second that. Detekt should ignore such cases.
@lwasyl Thanks for bringing this up.
